src/VM/OpcodeHandler.h

```diff
@@ -27,6 +27,9 @@
 
 // Third party includes
 
+// Marks handler parameter as unused and prevents compiler warnings.
+#define UNUSED_ARG(arg) (void)(arg)
```

Comment:
Good point. I think we can just remove unused arguments and leave their names in comments